test/mockupdb/test_handshake.py

```diff
@@ -11,14 +11,56 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-
-from mockupdb import MockupDB, OpReply, OpMsg, absent, Command, go
+import mockupdb
+from mockupdb import (MockupDB, OpReply, OpMsg, OpMsgReply, OpQuery, absent,
+                      Command, go, Request)
 from pymongo import MongoClient, version as pymongo_version
 from pymongo.errors import OperationFailure
+from pymongo.server_api import ServerApi
 
 import unittest
-
+from copy import deepcopy
+import time
+
+def test_handshake_with_option(self, protocol, num_servers, **kwargs):
+    primary = MockupDB(verbose=False)
+    secondaries = [MockupDB(verbose=False) for _ in range(num_servers-1)]
```

Comment:
Is `verbose=False` needed? We never use it in any other tests.
Also since the handshake is the same for primaries and secondaries we can use a single MockupDB node instead of 10 servers.